R-Interface/jaspResults/src/jaspContainer.cpp

```diff
@@ -428,3 +475,25 @@ void jaspContainer::setError(std::string message)
 	_errorMessage = message;
 	setError();
 }
+
+void jaspContainer::renderPlotsOfChildren()
+{
+	for(auto & child : _data)
+		switch (child.second->getType())
+		{
+		case jaspObjectType::container:
+			static_cast<jaspContainer*>(child.second)->renderPlotsOfChildren();
+			break;
```

Comment:
A newline underneath each `break;` would have been nice, to let it breathe a little.